### PR TITLE
Add suggestions for updating the survey.

### PR DESCRIPTION
**Pull request by zhenqincn**

### CODE REVIEW

File: docs/awesome_llm_data.md [hunk 1]
```diff
@@ -220,8 +220,12 @@ These tags correspond to the taxonomy in our paper, and each work may be assigne
...
 |Panda-70m: Captioning 70m videos with multiple cross-modality teachers|![](https://img.shields.io/badge/Data4Model--Scaling--Up--Acquisition-f1db9d)|Table 2|
-|Multimodal C4: An Open, Billion-scale Corpus of Images Interleaved With Text|![](https://img.shields.io/badge/Data4Model--Scaling--Up--Acquisition-f1db9d)|Table 3|
+|Multimodal C4: An Open, Billion-scale Corpus of Images Interleaved With Text|![](https://img.shields.io/badge/Data4Model--Scaling--Up--Acquisition-f1db9d)|Table 2|
+|ChartAssisstant: A Universal Chart Multimodal Language Model via Chart-to-Table Pre-training and Multitask Instruction Tuning|![](https://img.shields.io/badge/Data4Model--Scaling--Up--Acquisition-f1db9d)|Table 2|
 
+# Contribution to This Survey
+Due to the rapid development in the field, this repository and our paper are continuously being updated and synchronized with each other. Please feel free to make pull requests or open issues to improve this list and add more related resources!
+You can add the titles of relevant papers to the table above, and provide suggested tags along with the corresponding sections if possible.
```
Comment (yxdyc) [inline docs/awesome_llm_data.md:228]: , and (*optionally*) provide suggested tags along with the corresponding sections if possible.

File: docs/awesome_llm_data.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Welcome to the "Awesome List" for data-model co-development of Multi-Modal Large
 ![Overview of Our Taxonomy](https://img.alicdn.com/imgextra/i1/O1CN01aN3TVo1mgGZAuSHJ4_!!6000000004983-2-tps-3255-1327.png)
 Soon we will provide a dynamic table of contents to help readers more easily navigate through the materials with features such as search, filter, and sort.
 
-Due to the rapid development in the field, this repository and our paper are continuously being updated and synchronized with each other. Please feel free to pull requests or open issues to improve this list and add more related resources!
+Due to the rapid development in the field, this repository and our paper are continuously being updated and synchronized with each other. Please feel free to make pull requests or open issues to improve this list and add more related resources!
```
Comment (yxdyc) [inline docs/awesome_llm_data.md:7]: improve --> [contribute to](the link of the bottom "Contribution to This Survey" section)

Comment (zhenqincn) [reply docs/awesome_llm_data.md]: This has been updated.

File: docs/awesome_llm_data.md (hunk 1 above)
Comment (zhenqincn) [reply docs/awesome_llm_data.md]: This has been updated.

**Review by yxdyc — APPROVED** [review]

**Pull request merged**